docs/docs/coverage/language/python.md

```diff
@@ -21,11 +21,11 @@ The following scanners are supported for Python packages.
 
 The following table provides an outline of the features Trivy offers.
 
-| Package manager | File             | Transitive dependencies | Dev dependencies | [Dependency graph][dependency-graph] | Position |
-|-----------------|------------------|:-----------------------:|:----------------:|:------------------------------------:|:--------:|
-| pip             | requirements.txt |            -            |     Include      |                  -                   |    ✓     |
-| Pipenv          | Pipfile.lock     |            ✓            |     Include      |                  -                   |    ✓     |
-| Poetry          | poetry.lock      |            ✓            |     Exclude      |                  ✓                   |    -     |
+| Package manager | File             | Transitive dependencies | Dev dependencies | [Dependency graph][dependency-graph] | Position | [Detection Priority][detection-priority] |
+|-----------------|------------------|:-----------------------:|:----------------:|:------------------------------------:|:--------:|:----------------------------------------:|
+| pip             | requirements.txt |            -            |     Include      |                  -                   |    ✓     |                    -                     |
```

Comment:
we can add table for conda `environment.yml` files (this file list `requirements.txt`)

Reply:
Did we forget to add `environment.yaml` to the doc?

Reply:
Oh, I found it under OS. Do you remember why we put it here?
https://aquasecurity.github.io/trivy/v0.54/docs/coverage/os/conda/

Reply:
`Conda` doesn't only contain `Python` packages.
You can install OS packages.
`Conda` is thus compilation of package manager (like `apt`) and `pip`.

I thought about adding `Conda` to package managers, but we don't have pages for that.
So we decided to put `Conda` in OS first.

Reply:
>You can install OS packages.

Is there any document about it?

Anyway, I added.
https://github.com/aquasecurity/trivy/pull/7288/commits/9e8cdb17910410bcf82cc726ef5d577c1e0c1862

Reply:
https://docs.anaconda.com/working-with-conda/packages/install-packages/#installing-conda-packages

e.g. you can install `curl` - https://anaconda.org/conda-forge/curl

Reply:
I have reconsidered this, but my personal definition of "OS packages" is a package distributed by an OS vendor, such as Red Hat and SUSE. Conda is simply building and distributing third-party software, which is different from an operating system. Bitnami does the same and it seems better to create a new category in addition to OS and Language. We should discuss it later.

Reply:
We just didn't have a suitable category.
But in general we can separate `conda` + `bitnami` into a separate category.